src/modules/nats/nats-container.ts

```diff
@@ -0,0 +1,99 @@
+import { GenericContainer } from "../../generic-container/generic-container";
+import { StartedTestContainer } from "../../test-container";
+import { RandomUuid } from "../../uuid";
+import { AbstractStartedContainer } from "../abstract-started-container";
+import { ConnectionOptions } from "nats/lib/nats-base-client/types";
```

Comment:
You should create your own type here, as we cannot include the `nats` library as a dependency